followers/serializers.py

```diff
@@ -0,0 +1,52 @@
+from rest_framework import serializers
+from .models import Followers
+from Users.models import CustomUser
+from django.contrib.auth import get_user_model

```

Comment:
if you are using the user model, there is no need to use the get_user_model function